frontend/src/tests/components/UCSBSubjectsTest/UCSBSubject.test.js

```diff
@@ -0,0 +1,144 @@
+import { render, waitFor, fireEvent } from '@testing-library/react';
```

Comment:
I tried to be consistent with UCSBDates so I named the folder and file to be:

UCSBSubjects
UCSBSubjectsForm.test.js